src/runtime/policy/ingress-validation.ts

```diff
@@ -70,11 +70,11 @@ export class IngressValidation {
   // - Otherwise the capabilities will be determined by capabilities of all the
   // Handles its data is derived from.
   private findHandleCapabilities(handle: Handle, capabilitiesByField: Map<string, Capabilities[]>, seenHandles = new Set<Handle>()): IngressValidationResult {
-    assert(handle.type.maybeEnsureResolved());
+    assert(handle.type.maybeEnsureResolved({restrictMaxBoundToEmpty: true}));
```

Comment:
restrictToMinBound instead of restrictMaxBoundToEmpty

Reply:
done!
and the good news - it actually helped me get rid of that painful piece of code we didn't like. yay! thanks.